routes.py

```diff
@@ -22,6 +22,8 @@
         handlers.api.ShakeHandler),
     (r"/api/(shake_name)/([a-zA-Z0-9\-]+)",
         handlers.api.ShakeHandler),
+    (r"/api/(shake_user)/([a-zA-Z0-9\-]+)",
```

Comment:
Usernames can also contain a `_` character, so the pattern should be `[a-zA-Z0-9\-\_]`.